columnflow/selection/cms/met_filters.py

```diff
@@ -83,7 +83,7 @@ def met_filters(
         # append to the result
         result = (result & vals)
 
-    return result
+    return result, SelectionResult()
```

Comment:
shouldn't this be the ak array itself? Result looks to be a boolean mask (which granted, is an ak Array itself, but it's fundamentally different than in other selectors, right?

```suggestion
    return result, SelectionResult(
        {
            "steps": {
                "met_filters": result,
            }
        }
    )
```

Reply:
Yup, you're a right. I did the same also for the json filter selector above.